.github/workflows/gradle.yaml

```
@@ -0,0 +1,41 @@
+name: Front Deployment
+
+on:
+  push:
+    branches:
+      - develop
```

```suggestion
on:
  push:
    branches:
      - main
```

main을 배포용으로 하기로 했기도 했고, 개발 진행중인 develop 브랜치에 배포 붙이면 자잘한 변화까지 너무 자주 배포될 것 같습니다!
대상 브랜치 main으로 바꾸면 좋을 것 같습니당

---

1. 보통은 devleop에서 cicd 검증을 통해 검증 완료된 코드라고 생각되면 main으로 올리는 과정을 채택하는 것으로 알고있습니다! main 브랜치의 안정성을 위해서 인 것으로 생각되는데, 수정한다면 다음과 같은 방법으로 수정해도 괜찮을까요??!
ci 자체가 개발하면서 지속적으로 체킹한다는 목적이 있으니, CI 관련한 부분은 develop, CD 관련한 부분은 main으로 수정하려 합니다!!

2. develop에 푸시하는 게 아닌 **PR 올릴 시에** 바로 액션이 돌아가도록 설정하는 게 편할까요? 이 경우에 ci가 자주 돌아간다는 단점이 있긴 한데 오류를 고치고 머지할 수 있습니다!
**푸시했을 때 액션 돌아가도록 하면(기존 방법)** hotfix 이슈를 새로 파서 작업 후 머지하는 과정이 있을 것 같아요

@wrryu09 @jeeminyi @seong-hui 의견 부탁드립니당

---

pr을 올릴 때 CI가 돌아가는 방법이 오류를 수정하고 머지할 수 있어 좋다고 생각합니다! 머지하고 CI에 오류가 생겨 다시 이슈를 파서 수정하는 것 보다는 전자가 좋을 것 같아요! 그리고 CD의 경우는 main에 한해서 하면 좋을 것 같습니다!

---

그렇다면 develop에 pr 올릴 시 ci 작동되게 해서 에러사항 있다면 수정하고, cd는 main 브랜치에서만 작동되도록 하는 게 좋을 것 같습니다! 꼼꼼하게 봐주셔서 감사합니다😄

---

CI 관련한 부분만 develop에 작동되도록 하는거 좋을 것 같습니다!